src/Compilers/CSharp/Portable/Symbols/MemberSignatureComparer.cs

```diff
@@ -155,12 +155,11 @@ internal class MemberSignatureComparer : IEqualityComparer<Symbol>
         /// <summary>
         /// This instance is used to check whether one property or event overrides another, according to the C# definition.
         /// <para>NOTE: C# ignores accessor member names.</para>
-        /// <para>CAVEAT: considers return types so that getters and setters will be treated the same.</para>
         /// </summary>
         public static readonly MemberSignatureComparer CSharpAccessorOverrideComparer = new MemberSignatureComparer(
```

Comment:
>CSharpAccessorOverrideComparer [](start = 55, length = 30)

Looking at the call sites that use this comparer, I see that they also use `RuntimeSignatureComparer` (depending on `accessorIsFromSomeCompilation`).
Should `RuntimeSignatureComparer` also be adjusted?

Reply:
We are not changing the runtime's rules for implicit overriding.  It includes the return type.  The language's concept doesn't.  So no change to the other comparer is needed.

The "relaxed" comparison is the C# language rules.  This is not a temporary PROTOTYPE.  It is the new rules.

---
In reply to: [413964273](https://github.com/dotnet/roslyn/pull/43576#discussion_r413964273) [](ancestors = 413964273)